sdk/core/Azure.Core/src/DelayStrategy.cs

```diff
@@ -0,0 +1,128 @@
+        /// <param name="factor"></param>
+        /// <param name="minJitterFactor"></param>
+        /// <param name="maxJitterFactor"></param>
+        /// <returns></returns>
+        public static DelayStrategy CreateExponentialDelayStrategy(
+            TimeSpan? initialDelay = default,
+            TimeSpan? maxDelay = default,
+            double factor = 2.0,
+            double minJitterFactor = 0.8,
+            double maxJitterFactor = 1.2)
+        {
+            return new ExponentialDelayStrategy(initialDelay ?? TimeSpan.FromSeconds(0.8), maxDelay ?? TimeSpan.FromMinutes(1), factor, minJitterFactor, maxJitterFactor);
+        }
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="delay"></param>
+        /// <param name="minJitterFactor"></param>
+        /// <param name="maxJitterFactor"></param>
+        /// <returns></returns>
+        public static DelayStrategy CreateFixedDelayStrategy(
+            TimeSpan? delay = default,
+            double minJitterFactor = 0.8,
+            double maxJitterFactor = 1.2)
+        {
+            return new FixedDelayStrategy(delay ?? TimeSpan.FromSeconds(0.8), minJitterFactor, maxJitterFactor);
+        }
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="sequence"></param>
+        /// <param name="minJitterFactor"></param>
+        /// <param name="maxJitterFactor"></param>
+        /// <returns></returns>
+        public static DelayStrategy CreateSequentialDelayStrategy(
+            IEnumerable<TimeSpan>? sequence = default,
+            double minJitterFactor = 0.8,
+            double maxJitterFactor = 1.2)
+        {
+            return new SequentialDelayStrategy(sequence, minJitterFactor, maxJitterFactor);
+        }
+
+        private TimeSpan ApplyJitter(TimeSpan delay)
+        {
+            return TimeSpan.FromMilliseconds(_random.Next((int)(delay.TotalMilliseconds * _minJitterFactor), (int)(delay.TotalMilliseconds * _maxJitterFactor)));
+        }
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="response"></param>
+        /// <param name="retryNumber"></param>
+        /// <returns></returns>
+        protected abstract TimeSpan GetNextDelayCore(Response? response, int retryNumber);
+
+        /// <summary>
+        /// Get the interval of next delay iteration.
+        /// </summary>
+        /// <remarks> Note that the value could change per call. </remarks>
+        /// <param name="response"> Server response. </param>
+        /// <param name="retryNumber"></param>
+        /// <param name="serverDelayHint"></param>
+        /// <param name="clientDelayHint"></param>
+        /// <returns> Delay interval of next iteration. </returns>
+        public virtual TimeSpan GetNextDelay(Response? response, int retryNumber, TimeSpan? serverDelayHint, TimeSpan? clientDelayHint)
+        {
+            return
+                Max(
```

Comment:
Should be min

Reply:
Remove virtual for now - can consider extensible options in the future if needed.